services/publicapi/test/get_transaction_test.go

```diff
@@ -42,6 +27,15 @@ func TestGetTransactionStatus_GetCommitStatusFromTxPool(t *testing.T) {
 			}).Build(),
 		})
 
+		// contract test
+		ok, errCalled := harness.txpMock.Verify()
```

Comment:
you could push all mock verifications to a single harness method `harness.verifyMocks()`